infra/gcp/terraform/modules/oci-proxy/main.tf

```
@@ -148,8 +147,8 @@ locals {
       environment_variables = [
         {
           name = "DEFAULT_AWS_BASE_URL",
-          // AWS eu-central-1 is Frankfurt
-          value = "https://prod-registry-k8s-io-eu-central-1.s3.dualstack.eu-central-1.amazonaws.com",
+          // AWS Cloudfront
+          value = "https://d39mqg4b1dx9z1.cloudfront.net1",
```

It feels like this `1` is not supposed to be here:
```suggestion
          value = "https://d39mqg4b1dx9z1.cloudfront.net",
```

---

Good catch! fixed.